draft-dcook-ppm-dap-interop-test-design.md

```diff
@@ -324,6 +352,24 @@ collector’s HPKE configuration for this task.
 {: title="Response JSON object structure"}
 
 
+### `/internal/test/add_authentication_token` {#collector-add-auth-token}
+
+Associate an authentication token with a task, for the purpose of making
+authenticated requests to the leader. Authenticated requests MUST include the
+token in an HTTP header named `DAP-Auth-Token`.
+
+|Key|Value|
+|`task_id`|A base64url-encoded DAP `TaskId`.|
+|`role`|The string `"collector"`.|
```

Comment:
What's the rationale for adding this field here, since its value will never be anything but "collector"?

Reply:
This keeps the request the same shape as those sent to the aggregator, for simplicity and code reuse. I'm also considering hoisting both these and `/internal/test/ready` into a non-role-specific section, to cut down on redundancy.

Reply:
Makes sense. We might also consider replacing the aggregatorId (0 or 1) in the add task endpoint with role ("leader" or "helper').

Reply:
I'll draft up a separate change to make role a string.